docs/release.md

```diff
@@ -0,0 +1,22 @@
+# Status AddOn Relesea process 
+
+## Steps to make release
+
+1. Checkout main and fetch from upstream
+2. git checkout <release branch> # e.g. release-0.2
+3. Rebase from main
```

Comment:
This looks very odd to me. Why would it be right to rebase an old release branch on `main`?

Reply:
The idea is to make a new release, and update the release branch with what we want to go into the release, most likely rebasing into the release branch.  I have updated the text to show the actual commands, but what would be your recommendation  here to keep the release branch current with what we want in the release?

Reply:
@pdettori: You have not pushed your updates yet, right?

I think first it should be clear whether or not there is an intent to support a notion of release branch and, if so, what it means.

I think that if there is to be a notion of a release branch then a release branch should correspond with a git branch. This would mean that there are two release procedures to document: starting a new release branch from `main`, and adding a patch release to an existing release branch (which is not necessarily the latest release branch).

Reply:
It's our decision to use or not release branches. From a quick survey in CNCF sandbox and incubating projects I noticed that more mature projects such as Argo, Crossplane etc. tend to use release branches, while less mature projects do not. If we think that at this point this is just additional overhead we can just tag a new release from main and not use release branches (which we can always decide to use in the future)